test/utils/image/manifest.go

```diff
@@ -232,7 +218,7 @@ func initImageConfigs() map[int]Config {
 	configs := map[int]Config{}
 	configs[CRDConversionWebhook] = Config{e2eRegistry, "crd-conversion-webhook", "1.13rev2"}
 	configs[AdmissionWebhook] = Config{e2eRegistry, "webhook", "1.15v1"}
-	configs[Agnhost] = Config{e2eRegistry, "agnhost", "1.0"}
+	configs[Agnhost] = Config{e2eRegistry, "agnhost", "2.0"}
```

Comment:
should this be `2.1` or does this need to be done as a followup?

Reply:
2.0.

It s set to 2.1 in part 2: https://github.com/kubernetes/kubernetes/pull/78396/files#diff-d511d08b2cdc3d78d9e75c3c131ed2ceR226